ipn/ipnlocal/local_test.go

```diff
@@ -729,6 +729,58 @@ func TestStatusWithoutPeers(t *testing.T) {
 	}
 }
 
+func TestStatusPeerCapabilities(t *testing.T) {
+	b := newTestLocalBackend(t)
+
+	var cc *mockControl
+	b.SetControlClientGetterForTesting(func(opts controlclient.Options) (controlclient.Client, error) {
+		cc = newClient(t, opts)
+
+		t.Logf("ccGen: new mockControl.")
+		cc.called("New")
+		return cc, nil
+	})
+	b.Start(ipn.Options{})
+	b.Login(nil)
+	b.setNetMapLocked(&netmap.NetworkMap{
+		SelfNode: (&tailcfg.Node{
+			MachineAuthorized: true,
+			Addresses:         ipps("100.101.101.101"),
+		}).View(),
+		Peers: []tailcfg.NodeView{
+			(&tailcfg.Node{
+				ID:              1,
+				StableID:        "foo",
+				IsWireGuardOnly: true,
+				Hostinfo:        (&tailcfg.Hostinfo{}).View(),
+				Capabilities:    []tailcfg.NodeCapability{tailcfg.CapabilitySSH},
+				CapMap: (tailcfg.NodeCapMap)(map[tailcfg.NodeCapability][]tailcfg.RawMessage{
+					tailcfg.CapabilitySSH: nil,
+				}),
+			}).View(),
+			(&tailcfg.Node{
+				ID:           2,
+				StableID:     "bar",
+				Hostinfo:     (&tailcfg.Hostinfo{}).View(),
+				Capabilities: []tailcfg.NodeCapability{tailcfg.CapabilitySSH},
```

Comment:
Having different capabilities between the two peers would be best, as well as another peer with no capabilities. One of the peers should have a capability that involves a slice in the CapMap.